packs/rabbitmq/sensors/queues_sensor.py

```diff
@@ -0,0 +1,46 @@
+from st2reactor.sensor.base import PollingSensor
+import pika
```

Comment:
Basically because Its what I played around and what I know how to use ;)

Tried to use Kombu some time ago for a work project, but there were some stoppers that we were unable to get over. Moving it to pika took a couple of days and there was no issues on anything since then and we have added more and more stuff.

So looks like a personal choice more than technical reasons to be honest.